requirements/dev.txt

```diff
@@ -2,3 +2,5 @@
 -r base.txt
 
 money-to-prisoners-common[testing]>=9.16,<9.17
+boto3>=1.11
+moto
```

Comment:
as above, i think moto isn't really necessary as there's no multi-step process using boto: just calling one method to upload a file can be mocked on the `upload_to_s3` function